tgui/packages/tgui/interfaces/HydrogenExchange.js

```diff
@@ -0,0 +1,70 @@
+import { useBackend } from '../backend';
+import { Button, LabeledList, Section } from '../components';
+import { Window } from '../layouts';
+
+export const HydrogenExchange = (props, context) => {
+  const { act, data } = useBackend(context);
+  const { credits, merits, next_merit_rate, credits_to_merits, credit_tax } =
+    data;
+  return (
+    <Window
+      title="Hydrogen Exchange"
+      theme="ntos_terminal"
+      width={400}
+      height={300}
+    >
+      <Window.Content>
+        <Section
```

Comment:
maybe make this and the credits section into a single section with the title "Stored Currencies", and merits/credits in a labeled list or something inside of it? using a section element then only using the title for information seems weird. or maybe nest the merits/credits sections inside of a larger one